pkgs/development/python-modules/labelbox/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, affine
+, attrs
+, certifi
+, chardet
+, click-plugins
+, click
+, cligj
+, idna
+, jinja2
+, markupsafe
+, numpy
+, pillow
+, pyparsing
+, rasterio
+, requests
+, shapely
+, snuggs
+, urllib3
+}:
+
+buildPythonPackage rec {
+  pname = "labelbox";
+  version = "2.0.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "63784e5ebb4193e9d7f2637d9c9cbe08ff7a25325d86e0c21d77cb613cc1ec4c";
+  };
+
+  propagatedBuildInputs = [requests ];
+
+  check = true;
```

Comment:
for buildPythonPackage, doCheck is defaulted to true, you can remove this
```suggestion
```

Reply:
Done.